WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/logic/interval/IntervalStatistics.java

```diff
@@ -159,7 +160,7 @@ private boolean intervalIsOlderThanThreshold(
 	}
 
 	private void calculateStatistics() {
```

Comment:
Maybe, but it will require some new wrappers for JUnit Interval and Execution, so I will leave that for just a little bit later.